bin/node/cli/src/service.rs

```diff
@@ -214,7 +207,7 @@ macro_rules! new_full {
 			name: Some(name),
 			observer_enabled: false,
 			keystore,
-			is_authority,
+			is_authority: role.is_authority(),
```

Comment:
That's a breaking change, right? Sentry node will now have `is_authority:false` here. Most likely it's ok, but just making sure.

Reply:
Fixed to restore the current behaviour